app/build.gradle

```diff
@@ -284,6 +289,12 @@ dependencies {
     // Date and time formatting
     implementation "org.ocpsoft.prettytime:prettytime:5.0.7.Final"
 
+    // Jetpack Compose
+    implementation(platform('androidx.compose:compose-bom:2024.02.01'))
```

Comment:
Nice idea to use the BOM :-)

Sooner or later we should probably switch to version catalogs (in a separate PR obviously)

Reply:
You're right, migrating to version catalogs would be a good next step. This will track the migration https://github.com/TeamNewPipe/NewPipe/issues/10927